app/src/main/java/com/kickstarter/viewmodels/ThreadViewModel.kt

```diff
@@ -22,6 +28,10 @@ interface ThreadViewModel {
 
         /** Will tell to the compose view if should open the keyboard */
         fun shouldFocusOnCompose(): Observable<Boolean>
```

Comment:
`shouldFocusOnCompose()` output was prepared already for showing/hidding the keyboard in case it's not necessary or the implementation has been done differently please remove

Reply:
![image](https://user-images.githubusercontent.com/1075310/122261072-49b63a80-ced4-11eb-9ea4-565a51b65546.png)